src/layouts/components/Header.tsx

```diff
@@ -465,21 +486,21 @@ function ComingSoonTag() {
 
 enum MobileTabletDropDownState {
   EXPLORE = "Explore",
-  ECOSYSTEM = "Ecosystem",
   BUILD = "Build",
+  METACHAIN = "Meta Chain",
   COMMUNITY = "Community",
 }
 
 const MenuItems = [
   MobileTabletDropDownState.EXPLORE,
-  MobileTabletDropDownState.ECOSYSTEM,
   MobileTabletDropDownState.BUILD,
+  MobileTabletDropDownState.METACHAIN,
   MobileTabletDropDownState.COMMUNITY,
 ];
 
 const dropDownMapping = {
   explore: Explore,
-  ecosystem: Ecosystem,
+  // ecosystem: Ecosystem,
```

Comment:
curious why we commented this one but removed at other places

Reply:
removed it ! Initially, i thought we might have to use the ecosystem component in the future

Reply:
maybe check with gracia for this?